src/apps/backend/modules/comment/internal/comment_writer.py

```diff
@@ -0,0 +1,46 @@
+from datetime import datetime, timezone
+
+from bson import ObjectId
+
+from modules.comment.errors import CommentCreationError, CommentUpdateError
+from modules.comment.internal.comment_util import validate_task_and_user_exists
+from modules.comment.internal.store.comment_model import CommentModel
+from modules.comment.internal.store.comment_repository import CommentRepository
+from modules.comment.types import Comment, CreateCommentParams
+
+
+class CommentWriter:
+    @staticmethod
+    def create_comment(*, params: CreateCommentParams) -> Comment:
+        try:
+            validate_task_and_user_exists(params.task_id, params.user_id)
+            comment_model = CommentModel(
+                id=None, task_id=params.task_id, user_id=params.user_id, content=params.content
+            )
+
+            comment_bson = comment_model.to_bson()
```

Comment:
Ideally, this should not be needed.

`CommentRepository.collection().insert_one` should be called directly via params